tests/test_subber.py

```diff
@@ -19,7 +19,10 @@ def test_get_form(self):
         self.assert_template_used('form.html')
 
     @patch('subber.reddit.get_user_recommendations')
-    def test_get_sub_recommendations(self, mock_recommendations):
+    @patch('subber.subber.session')
```

Comment:
Since the session is patched in all of the individual functions with this PR, can you remove line 12 and move line 13 by the other import statements?

Reply:
No problem! Thanks.